frontend/src/views/Layout/DashboardLayout_gdd.spec.js

```diff
@@ -222,6 +223,65 @@ describe('DashboardLayoutGdd', () => {
           expect(toasterMock).toBeCalledWith('Ouch!')
         })
       })
+
+      describe('set visible method', () => {
+        beforeEach(() => {
+          wrapper.findComponent({ name: 'Navbar' }).vm.$emit('set-visible', true)
+        })
+
+        it('sets visible to true', () => {
+          expect(wrapper.vm.visible).toBe(true)
+        })
+      })
+
+      describe('elopage URI', () => {
+        describe('user has no publisher ID and no elopage', () => {
+          beforeEach(() => {
+            mocks.$store.state.publisherId = null
+            mocks.$store.state.hasElopage = false
+            wrapper = Wrapper()
+          })
+
+          it('links to basic-de', () => {
+            expect(wrapper.vm.elopageUri).toBe(
+              'https://elopage.com/s/gradido/basic-de/payment?locale=en&prid=111&pid=2896&firstName=User&lastName=Example&email=user@example.org',
```

Comment:
`basic-de` but `?locale=en` ; do I miss something?

Reply:
I have no idea. I just copied the link from the code. Can we check this?

Reply:
The link is working. Only German is supported on the target page